### DISCUSSION TITLE
Editor Window Split?

### DISCUSSION THREAD

**Discussion by BobPstats** [texstudio-org/texstudio#3039] [Q&A]
I don't know what I did, but my window is now split and I don't know what the right is for. How do I get rid of that right pane? I'm on v4.5.1. 

![image](https://user-images.githubusercontent.com/128639146/227018447-332bea17-6268-4774-a1ad-60769fd6dc59.png)

**Comment by octaeder** [comment]
Please check in config (options menu) following setting:

![grafik](https://user-images.githubusercontent.com/102688820/227030223-f70bf370-9b62-4a3b-965f-ad04c80e5684.png)

**Reply by BobPstats** [reply]
Thank you! That makes sense.

I was looking through config to find out how to turn off the auto updating in the PDF viewer, and I must have accidentally bumped that.

**Comment by octaeder** [comment]
did you scroll with a mouse wheel in the config?

**Reply by BobPstats** [reply]
Yeah probably so. Texstudio is super laggy recently, and I was trying to think of features I could disable to help, like that auto updating.
